src/asynk/async_queue.rs

```diff
@@ -573,47 +515,33 @@ where
         }
     }
 
-    fn row_to_periodic_task(row: Row) -> PeriodicTask {
-        let id: Uuid = row.get("id");
-        let metadata: serde_json::Value = row.get("metadata");
-        let period_in_millis: i64 = row.get("period_in_millis");
-        let scheduled_at: Option<DateTime<Utc>> = match row.try_get("scheduled_at") {
-            Ok(datetime) => Some(datetime),
-            Err(_) => None,
-        };
-        let created_at: DateTime<Utc> = row.get("created_at");
-        let updated_at: DateTime<Utc> = row.get("updated_at");
-
-        PeriodicTask::builder()
-            .id(id)
-            .metadata(metadata)
-            .period_in_millis(period_in_millis)
-            .scheduled_at(scheduled_at)
-            .created_at(created_at)
-            .updated_at(updated_at)
-            .build()
-    }
-
     fn row_to_task(row: Row) -> Task {
         let id: Uuid = row.get("id");
         let metadata: serde_json::Value = row.get("metadata");
         let error_message: Option<String> = match row.try_get("error_message") {
             Ok(error_message) => Some(error_message),
             Err(_) => None,
         };
+        let uniq_hash: Option<String> = match row.try_get("uniq_hash") {
```

Comment:
I think it can be replaced with  `row.try_get("uniq_hash").ok()`

Reply:
[comments suggestions](https://github.com/ayrat555/fang/pull/71/commits/4eb86a2e2a6a2e33153736332de9191e4a162d56)